packages/suite/src/views/wallet/coinmarket/savings/bank-account/index.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Translation } from '@suite-components';
+import { Button, Input, Select } from '@trezor/components';
+import { InputError, KYCInProgress, withCoinmarket, WithCoinmarketProps } from '@wallet-components';
+import { getInputState } from '@wallet-utils/sendFormUtils';
+import { useCoinmarketSavingsBankAccount } from '@wallet-hooks/useCoinmarketSavingsBankAccount';
+import { Controller } from 'react-hook-form';
+import { UsBankAccountTypes } from '@wallet-constants/coinmarket/savings';
+import { buildUsBankAccountType } from '@wallet-utils/coinmarket/coinmarketUtils';
+
+const Header = styled.div`
+    font-size: 24px;
+    line-height: 24px;
+    margin-bottom: 31px;
+`;
+
+const BankAccountTypeNote = styled.div`
+    font-size: 14px;
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
```

Comment:
Please, destructure props for better readability ;)

```suggestion
    color: ${({theme}) => theme.TYPE_LIGHT_GREY};
```